datasets/spanish_dish_title/src/scraper_images_dish.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+import newspaper as ns
+from bs4 import BeautifulSoup
+import requests
+from pathlib import Path
+
+
+def donwload(url, directory):
```

Comment:
Por favor incluye una pequeña descripción de las funciones en docstrings, p.ej:

```suggestion
def donwload(url, directory):
    """
    Descargar ...
    """
```